src/Codeception/Subscriber/ErrorHandler.php

```diff
@@ -39,11 +44,16 @@ public function handle(SuiteEvent $e)
             $this->errorLevel = eval("return {$settings['error_level']};");
         }
         error_reporting($this->errorLevel);
+
+        if (self::$initialized) {
```

Comment:
`set_error_handler()` acts globally, and I think a flag related with it also must be "global".
I just fixed a work-stopping issue for myself and appreciate a more sophisticated solution.